src/tmpl/blm.pug

```diff
@@ -0,0 +1,47 @@
+doctype html
```

Comment:
@isaacdurazo Could you use the existing layout? Like this one for example https://github.com/gruntjs/gruntjs.com/blob/b89770f12c5c405cb46f2dc21491615fa26f6cc7/src/tmpl/plugins.pug ?

Reply:
@vladikoff I actually tried to do that, but the problem I was having is that 1) the banner at the top was getting included and 2) I needed to modify the nav, specifically the logo as you can see in the screenshot. 

It's my first time using Pug and tried to find a way to do it with a conditional statement or something like that, but couldn't find a good solution. Do you have a suggestion for a workaround?